native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -347,60 +564,45 @@ EMSCRIPTEN_BINDINGS(spine) {
     class_<Interpolation>("Interpolation")
         .function("apply", &Interpolation::apply, pure_virtual());
 
-    class_<Triangulator>("Triangulator")
-        .constructor<>()
-        .function("triangulate", &Triangulator::triangulate)
-        .function("decompose", &Triangulator::decompose, allow_raw_pointers());
+    class_<HasRendererObject>("HasRendererObject")
+        .constructor<>();
+
+    // class_<Triangulator>("Triangulator")
+    //     .constructor<>()
+    //     .function("triangulate", &Triangulator::triangulate)
+    //     .function("decompose", &Triangulator::decompose, allow_raw_pointers());
```

Comment:
Doesn't need to bind `Triangulator` since it's only used in `SkeletonClipping` class and it's a private property of SkeletonClipping.